### PR TITLE
Minor man page improvement

### PR DESCRIPTION
**Pull request by dzafman**

### CODE REVIEW

File: doc/man/8/ceph.rst [hunk 1]
```diff
@@ -841,14 +841,18 @@ Subcommand ``get`` gets pool parameter <var>.
...
+	ceph osd pool get hit_set_type|hit_set_period|hit_set_count|hit_set_fpp|
+	target_max_objects|target_max_bytes|cache_target_dirty_ratio|
+	cache_target_full_ratio|cache_min_flush_age|cache_min_evict_age|
+	min_read_recency_for_promote
 
-	ceph osd pool get <poolname> cache_min_flush_age|cache_min_evict_age|
-	erasure_code_profile|min_read_recency_for_promote|write_fadvise_dontneed
+Only for erasure coded pools::
+
+	ceph osd pool erasure_code_profile
```
Comment (dachary) [inline doc/man/8/ceph.rst]: <pre>
ceph osd pool get &lt;poolname> erasure_code_profile
</pre>

File: doc/man/8/ceph.rst [hunk 2]
```diff
@@ -841,14 +841,18 @@ Subcommand ``get`` gets pool parameter <var>.
 Usage::
 
 	ceph osd pool get <poolname> size|min_size|crash_replay_interval|pg_num|
-	pgp_num|crush_ruleset|hit_set_type|hit_set_period|hit_set_count|hit_set_fpp
+	pgp_num|crush_ruleset|auid|write_fadvise_dontneed
 
-	ceph osd pool get <poolname> auid|target_max_objects|target_max_bytes
+Only for tiered pools::
 
-	ceph osd pool get <poolname> cache_target_dirty_ratio|cache_target_full_ratio
```
Comment (dachary) [inline doc/man/8/ceph.rst]: <pre>
ceph osd pool get &lt;poolname>
</pre>

**Comment by dachary** [comment]
splitting the **ceph osd pool get <poolname>** help in three parts makes it easier to read, good idea :-)

**Comment by dzafman** [comment]
@dachary Fixed two issues.  Also, regenerated ceph.8 file.

**Comment by dzafman** [comment]
@dachary Ooops, need to fix again.

**Pull request merged**